odoo/tools/query.py

```diff
@@ -178,10 +187,11 @@ def select(self, *args: str | SQL) -> SQL:
         """ Return the SELECT query as an ``SQL`` object. """
         sql_args = map(SQL, args) if args else [SQL.identifier(self.table, 'id')]
         return SQL(
-            "%s%s%s%s%s%s",
+            "%s%s%s%s%s%s%s",
             SQL("SELECT %s", SQL(", ").join(sql_args)),
             SQL(" FROM %s", self.from_clause),
             SQL(" WHERE %s", self.where_clause) if self._where_clauses else SQL(),
+            SQL(" GROUP BY %s", self._groupby) if self._groupby else SQL(),
```

Comment:
By default, we return "select id from ...", but this does not make sense in case of group by in queries. Maybe we should specify raise an error if we try to select without specifying what we want.

Reply:
I dont' think it is necessary, `Query.select` doesn't have to check the input of `args`. For example, if the developer tries to select a non-existing column, we also don't raise a special error but use the pyscopg error.
Also if it happens, it should be a coding error instead of bad input data from the client. And the error message from psycopg should be clear enough for debugging purpose.